setup.py

```diff
@@ -11,14 +11,33 @@
     "tensorflow>=2.4.0",
 ]
 
+with open("README.md", "r") as file:
+    long_description = file.read()
+
+with open("VERSION", "r") as file:
+    version = file.read()
+
 setup(
     name="gpflux",
-    version="0.1",
+    version=version,
     author="Secondmind Labs",
     author_email="gpflux@secondmind.ai",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
     description="GPflux: Deep GP library",
+    license="Apache License 2.0",
     keywords="Deep-Gaussian-processes",
     install_requires=requirements,
     packages=find_namespace_packages(include=["gpflux*"]),
     package_data={"gpflux": ["py.typed"]},
+    project_urls={
+        "Source on GitHub": "https://github.com/secondmind-labs/GPflux",
+        "Documentation": "https://secondmind-labs.github.io/GPflux/",
+    },
+    classifiers=[
+        "License :: OSI Approved :: Apache Software License",
+        "Programming Language :: Python :: 3.7",
```

Comment:
should this include 3.8 as well ?

Reply:
NB- what about 3.6 (still the default on plenty of current LTS distributions, e.g. RedHat I believe)? we could easily add it to the CI matrix (and drop it if it restricts us too much - but I'm not aware of using any 3.7-specific features).

Reply:
Updated the classifiers with Python 3.6 and 3.8. Can you please check I correctly added python 3.6 to the CI matrix?